contributed/Source+Catalog+Emulation+with+CosmoDC2Realizer.ipynb

```diff
@@ -0,0 +1,535 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Where we'll store our data files\n",
+    "data_dir = 'data'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download OpSim db and gunzip\n",
+    "# Running this cell is not necessary.\n",
+    "# If you want to but it doesn't run, simply run it in the terminal.\n",
+    "!mkdir data\n",
+    "!cd data\n",
+    "!wget opsim.lsst.org/runs/minion_1016/data/minion_1016_sqlite.db.gz\n",
+    "!gunzip -k minion_1016_sqlite.db.gz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[('Session',), ('Config',), ('Field',), ('ObsHistory',), ('Proposal',), ('SeqHistory',), ('SlewHistory',), ('SlewActivities',), ('SlewState',), ('SlewMaxSpeeds',), ('TimeHistory',), ('ObsHistory_Proposal',), ('Cloud',), ('Seeing',), ('Log',), ('Config_File',), ('Proposal_Field',), ('SeqHistory_ObsHistory',), ('MissedHistory',), ('SeqHistory_MissedHistory',), ('Summary',)]\n",
+      "CPU times: user 1.21 ms, sys: 781 µs, total: 1.99 ms\n",
+      "Wall time: 28.4 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "# Read in the minion_1016 opsim database\n",
+    "opsim_v3 = os.path.join(data_dir, 'minion_1016_sqlite.db')\n",
+    "conn = sqlite3.connect(opsim_v3)\n",
+    "\n",
+    "# See which tables the db file has\n",
+    "cursor = conn.cursor()\n",
+    "cursor.execute(\"SELECT name FROM sqlite_master WHERE type='table';\")\n",
+    "print(cursor.fetchall())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are primarily interested in two tables of the `minion_1016` database: `ObsHistory` containing the observation conditions and `Field` containing the field positions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 30.8 s, sys: 5.74 s, total: 36.5 s\n",
+      "Wall time: 36.6 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time \n",
+    "# ~ 40s in Jupyter-dev\n",
+    "# Save the tables ObsHistory and Field as Pandas DataFrames\n",
+    "obs_history = pd.read_sql(sql='SELECT * from ObsHistory', con=conn)\n",
+    "field = pd.read_sql(sql='SELECT * from Field', con=conn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For speed considerations, we will only work with galaxies in `cosmoDC2_v1.0_9556`, a version of the extragalactic catalog restricted to one healpixel. This healpixel, it turns out, roughly coincides with the OpSim field with ID 1188 so we pre-save a subset of the `ObsHistory` table with the columns we'll need."
+   ]
+  },
```

Comment:
It might be good to add a function/call with the full set, in case folks want to run the larger case, but have it commented out for the demo run?

Reply:
Done (in the GCR query, as the truth catalog query only depends on the sampling positions and radius)!